woocommerce/woocommerce-ios
# Adding more menu button in product variations

Closes: #6238
Merge after #6247

### Description
Adds a new menu button in the navigation bar of the product variations list, that shows an action sheet with the "Cancel" and "Bulk Update" options. The "Bulk Update" will latter trigger an view in #6239.

### Testing instructions
1.  Go to Products
2. Select a product with variations
3. Select variations
4. In the top right there should exist a "more" button
5. Tapping the "more" button an action sheet is displayed
6. Tapping the "cancel" button dismisses the action sheet

### Screenshots
![Simulator Screen Shot - iPod touch (7th generation) - 2022-02-19 at 10 30 51](https://user-images.githubusercontent.com/96764631/154850194-3a607577-d7dd-438c-b5f4-bcf84061952e.png)

---
